component/testrunner/testrunner.json

```diff
@@ -161,7 +161,7 @@
         },
         "uris" :
         {
-          "add-nocache-param" : true
+          "add-nocache-param" : false
```

Comment:
Well, as I said, it's hard enough to get tests to refresh properly even with that option enabled. I'm not eager to make this particular (really unrelated to the PR) change, although I have no problems with the remainder of the PR.